src/app/core/layout/header/header.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterLink } from '@angular/router';
+import { AuthenticationService } from '../../services/authentication.service';
+import { MmsaiAccountButtonComponent } from '../../components/mmsai-account-button/mmsai-account-button.component';
+import { MenuItem } from 'primeng/api';
+import { Store } from '@ngrx/store';
+import { signOut } from '../../states/auth/auth.actions';
+
+@Component({
+  selector: 'app-header',
+  standalone: true,
+  imports: [CommonModule, RouterLink, MmsaiAccountButtonComponent],
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss'],
+})
+export class HeaderComponent implements OnInit {
+  isLoggedIn!: boolean;
+  displayName?: string | null;
```

Comment:
In this case I likely added it because eslint recommended it. I'll go through the components and unify the usage of the operators and put more thought into assertions in the future.